docs/other.rst

```diff
@@ -79,9 +79,17 @@ node is not in a node list. A node list will never have a :file:`.next` or
 Nodes also have a :file:`.next_generator()` and :file:`.previous_generator()`
 if you want to iterate on the neighbours of the node.
 
+Nodes have a :file:`.next_recursive` and :file:`.previous_recursive`
+attribute that point to the next or previous node if the node is located
+in a node list. They are set at :file:`None` if the node is not in
+a node list but, compared to the simple :file:`.next` and
+:file:`.previous` attributes, they are not set to :file:`None` if there
+are no adjacent nodes. In that case, they point to the first adjacent
+node that exist in the parent hierarchy.
```

Comment:
I don't find this sentence very clear "In that case, they point to the first adjacent
node that exist in the parent hierarchy." or at least I'm not expecting people to really understand it easily :/